docs/index.md

```diff
@@ -0,0 +1,84 @@
+# GeoTrellis Documention
+
+## What is GeoTrellis?
+
+*GeoTrellis* is a Scala library and framework that uses
+Spark to work with raster data.  It is released under
+the Apache 2 License.
+
+GeoTrellis reads, writes, and operates on raster data
+as fast as possible. It implements many
+[Map Algebra](http://en.wikipedia.org/wiki/Map_algebra)
+operations as well as vector to raster or raster to
+vector operations.
+
+GeoTrellis also provides tools to render rasters into
+PNGs or to store metadata about raster files as JSON.
+It aims to provide raster processing at web speeds (sub-second
+or less) with RESTful endpoints as well as provide
+fast batch processing of large raster data sets.
+
+Please visit the **[project site](http://geotrellis.io)**
+for more information as well as some interactive demos.
+
+
+## Why GeoTrellis?
+	
+## Contact and Support
+
+You can find more information and talk to developers
+(let us know what you're working on!) at:
+
+  - [Gitter](https://gitter.im/geotrellis/geotrellis)
+  - [GeoTrellis mailing list](https://groups.google.com/group/geotrellis-user)
+
+
+
+
+## Hello Raster!
+
```

Comment:
Let's revisit if we should keep this example in this file. Open question: do we want to be showing any code in the first thing they see when they visit our docs?